MAINTAINERS.md

```diff
@@ -10,4 +10,6 @@ The current Maintainers Group for the Kubevirt Project consists of:
 | [Vasiliy Ulyanov](https://github.com/vasiliy-ul) | SuSE | |
 | [Chris Calligari](https://github.com/mazzystr) | Red Hat | |
 
+This list must be kept in sync with the [CNCF Project Maintainers list](https://github.com/cncf/foundation/blob/master/project-maintainers.csv).
```

Comment:
Just a thought, having the list in the same order in both places would help with sync

Reply:
CNCF has automation that sync's the csv